.travis.yml

```diff
@@ -6,7 +6,6 @@ branches:
 python:
  - "3.6"
  - "3.5"
- - "3.4"
 env:
  - DJANGO_VERSION=1.8.17
  - DJANGO_VERSION=1.10.5
```

Comment:
Good catch--I can't add 1.11 just yet because we have some issues w/ it (see #21) but I will definitely update the patch versions for the ones we do support.